geometry_msgs/PoseStamped pose    # bounding box pose (with respect to center)
geometry_msgs/Vector3 dimensions  # x, y, z dimensions (with respect to coordinate frame and rotation defined in pose)